app/assets/stylesheets/application.scss

```diff
@@ -1,244 +1,68 @@
-@import "measurements";
-@import "typography";
-@import "colours";
-@import "reset";
-@import "shims";
+$govuk-compatibility-govuktemplate: true;
 @import "govuk_publishing_components/all_components";
 
 .info-frontend {
-  margin-bottom: $gutter;
-  @include media(tablet){
-    padding-top: $gutter-half;
-    margin-bottom: $gutter * 2;
-  }
-  @extend %contain-floats;
-
-  hr {
-    margin: 0 $gutter-half;
-    border: 1px solid $border-colour;
-    border-width: 2px 0 0;
-    display: block;
-    clear: both;
-  }
-
-  .padding-for-tablets {
-    @include media(tablet){
-      padding: $gutter-half;
-      padding-left: 0;
-    }
-  }
-
-  .time-interval-explanation {
-    margin-top: $gutter-one-third;
-    color: $secondary-text-colour;
-  }
-
-  h2 {
-    @include bold-36;
-    display: block;
-    margin-bottom: $gutter-one-third;;
-  }
-
-  table {
-    border-collapse: collapse;
-    border-spacing: 0;
-    margin: 1em 0 2em;
-    width: 100%;
-
-    caption {
-      @include bold-24;
-      margin: 0.4em 0.5em 0.4em 0.7em;
-      text-align: left;
-    }
-
-    @media screen and (max-width: 320px) {
-      margin: 1em 0 2em 0;
-      width: 100%;
-
-      caption {
-        padding-left: 0;
-      }
-    }
-  }
-
-  th, td {
-    @include core-14;
-    vertical-align: top;
-    padding: 0.7em 0.5em 0.7em 0;
-  }
-
-  td {
-    border-bottom: solid 1px $border-colour;
-
-    small {
-      @include core-16;
-    }
-  }
-
-  /* reduce table font size on very small screens */
-  @media screen and (max-width: 320px) {
-    th, td, td small {
-      font-size: 10px;
-    }
-  }
-
-  th {
-    line-height: 1.25em;
-    text-align: left;
-    font-weight: 700;
-    border-bottom: solid 1px $border-colour;
-  }
-
-  .metrics-panel {
-    h2 {
-      @include bold-24;
-      display: block;
-      margin-top: -5px;
-      margin-bottom: $gutter-one-third;
-    }
-
-    @extend %contain-floats;
-    clear: both;
-    @include media(tablet){
-      padding-bottom: $gutter-two-thirds;
-    }
-
-    .no-metrics {
-      @include bold-27;
-    }
-
-    .count {
-      @include bold-48;
-    }
-    .unit {
-      @include core-24;
-    }
-  }
-
-  .lead-metric {
```

Comment:
I think this needs to be replaced somehow:

### Before
<img width="575" alt="Screenshot at Sep 27 11-07-36 pm" src="https://user-images.githubusercontent.com/424772/65804931-9a738600-e17b-11e9-84ef-25017e999ef4.png">


### After
<img width="1108" alt="Screenshot at Sep 27 11-06-06 pm" src="https://user-images.githubusercontent.com/424772/65804906-7e6fe480-e17b-11e9-93ae-0891b4b59cd9.png">

Reply:
nice spot. fixed now

Before:
<img width="346" alt="Screen Shot 2019-09-28 at 13 33 55" src="https://user-images.githubusercontent.com/3758555/65816539-d991ed80-e1f4-11e9-82c8-d913aac5b9d4.png">

After:
<img width="368" alt="Screen Shot 2019-09-28 at 13 35 18" src="https://user-images.githubusercontent.com/3758555/65816540-df87ce80-e1f4-11e9-97ca-89268db7234e.png">